doc/source/reference/command-line.rst

```diff
@@ -208,29 +197,50 @@ command-line.
 rpdb
 ++++
 
-Connect to a currently running (on localhost) rpd session.
+Connect to a currently running (on localhost) rpd session. See
+:ref:`remote-debugging` for more info.
 
 notebook
 ++++++++
 
-Launches an IPython notebook server and prints out instructions on how to open
-an ssh tunnel to connect to the notebook server with a web browser.  This is
-most useful when you want to run an IPython notebook using CPUs on a remote
+Launches a Jupyter notebook server and prints out instructions on how to open
+an ssh tunnel to connect to the notebook server with a web browser. This is
+most useful when you want to run a Jupyter notebook using CPUs on a remote
 host.
 
 stats
 +++++
 
 This subcommand provides you with some basic statistics on a given dataset.
 It provides you with the number of grids and cells in each level, the time
-of the dataset, the resolution, and the maximum density in a variety of units.
-It is tantamount to performing the ``print_stats()`` inside of yt.
+of the dataset, and the resolution. It is tantamount to performing the
+``print_stats()`` inside of yt.
+
+Additionally, there is the option to print the minimum, maximum, or both for
+a given field. The field is assumed to be density by default:
+
+.. code-block:: bash
+
+   yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max --min
+
+or a different field can be specified using the ``-f`` flag:
+
+.. code-block:: bash
+
+   yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max --min -f temperature
```

Comment:
just curious, does this comment support ftype, fname tuples ?

Reply:
Yes, if you specify them like this:
```bash
yt stats GasSloshing/sloshing_nomag2_hdf5_plt_cnt_0150 --max --min -f gas,temperature
```

Reply:
ok cool. In this case I'd argue that it's probably best to use it here, so we don't create a bug if we ever decide to drop support for single key field (not sure we should, but it's becoming a possibly lately).

Reply:
done.